features/admin.users.v1/components/user-impersonation-action.tsx

```diff
@@ -442,7 +457,7 @@ export const UserImpersonationAction: FunctionComponent<UserImpersonationActionI
     const resolveUserActions = (): ReactElement => {
 
         return (
-            !isSubOrgUser && !isUserCurrentLoggedInUser
+            !isSubOrgUser && !isUserCurrentLoggedInUser && isInTheSameUserstore()
```

Comment:
Wouldn't it be better if we show some kind of info message when the user stores don't match? otherwise, some admins may be confused why impersonate action is only available for some users.

Reply:
If the button is shown in a disabled state, it implies that the user can take some action to enable it. But if there's nothing the user can do to enable it, is it necessary to show the button at all?

Reply:
We can show a disabled button with a hint that explains the situation, IMO. Otherwise there will be queries on why impersonate is only enabled for some users. (or cover these in docs maybe)